### PR TITLE
Refactor/structure

### PR DESCRIPTION
**Pull request by davecarter**
## PR Summary

This PR adds a custom classname to the `SuiCard` generic `div` wrapper. This feature is quite useful when many Cards are contained inside the same wrapper and needs to have different classnames each.

Is also useful when applying different Flexbox property `flex-basis` values to multiple Cards.
If any custom class is provided the container will have the default `sui-Card` class.
In addition, includes the following improvements:

- Now `primary-content`is required.
- If `secondary-content` is not provided nothing is returned
- Updated README.md file

Please review @nucliweb @kikoruiz @ruben-martin-lozano @danderu

### CODE REVIEW

File: src/sui-card/index.jsx [hunk 1]
```diff
@@ -31,9 +33,10 @@ class SuiCard extends React.Component {
         <div className='sui-Card-primary'>
           {this.props.primary}
         </div>
-        <div className='sui-Card-secondary'>
+        {this.props.secondary &&
```
Comment (kikoruiz) [inline src/sui-card/index.jsx]: :clap:

File: docs/index.jsx [hunk 2]
```diff
@@ -11,24 +10,24 @@ import './styles/codemirror.scss';
...
-}];
+};
 
 ReactDom.render(
-    <SuiCard
-      primary={
-        <SuiMultimedia images={images} />
-      }
-      secondary={
+    <SuiCard className={'myCustom-class'}
```
Comment (kikoruiz) [inline docs/index.jsx]: I would prefer to show different use cases of Sui-Card here, what do you think?

File: README.md [hunk 3]
```diff
@@ -9,17 +9,46 @@ The SUI-Card component features a placeholder to portray a custom primary and a
...
 
 Check out this example below
 ```javascript
 
 ReactDom.render(
     <SuiCard
-      primary={
-        <SuiMultimedia images={data} />
+      primary={[
+        <a href={data.link}><img src={data.src} /></a>
```
Comment (ruben-martin-lozano) [inline README.md]: Why don't we use SuiMultimedia here??

File: docs/index.jsx (hunk 2 above)
Comment (ruben-martin-lozano) [inline docs/index.jsx]: +1000

Comment (davecarter) [inline docs/index.jsx]: Agree :octocat:

File: README.md (hunk 3 above)
Comment (davecarter) [inline README.md]: I'm going to add a new use case to show how to implement a SuiMultimedia component

**Comment by kikoruiz** [comment]
:+1:

**Comment by aleanez** [comment]
:+1:

**Pull request merged**